### PR TITLE
Convert to SQLite+S3 backend rather than a dedicated PostgreSQL database

### PR DESCRIPTION
**Pull request by colebrumley**
We are migrating this application to a new hosting platform and no longer want to run it with a dedicated PostgreSQL database to streamline maintenance and operations. Since we typically don't run this application at large horizontal scale, I've refactored it to leverage a local SQLite database that gets pulled from S3 (or created) on startup, and pushed to S3 on shutdown.

I also fleshed-out the readme a bit more with some of the new env variables I added and some more ELI5 running instructions.

### CODE REVIEW

File: backend/db.js [hunk 1]
```diff
@@ -1,47 +1,24 @@
-const Sequelize = require("sequelize");
+const sqlite3 = require('sqlite3').verbose();
```
Comment (tgriesser) [inline backend/db.js:1]: Just wanted to mention we have been using https://github.com/WiseLibs/better-sqlite3 in other places where we're using sql. 

Don't necessarily need to change things here, but if you want to keep it consistent, it's convenient b/c the API is synchronous

**Review by tgriesser — APPROVED** [review]
Nice! Didn't pull it down to try it but it LGTM

Just added one comment, but if it works I think it's good to use as is

File: backend/db.js (hunk 1 above)
Comment (colebrumley) [reply backend/db.js]: I went ahead and swapped them out
